src/vm/functions/mod.rs

```diff
@@ -283,6 +287,30 @@ pub fn lookup_reserved_functions(name: &str, version: &ClarityVersion) -> Option
                 // TODO: Create a dedicated cost function for this case.
                 ClarityCostFunction::Mul,
             ),
+            StringToInt => NativeFunction(
+                "native_string_to_int",
+                NativeHandle::SingleArg(&conversions::native_string_to_int),
+                // TODO: Create a dedicated cost function for this case.
+                ClarityCostFunction::Mul,
```

Comment:
I wonder if it would be worth creating an `Unimplemented` cost function enum member to make it easy to keep track of the cost functions we will need to implement before 2.1

Reply:
This could be useful, I can open a PR that adds the `Unimplemented` enum member pretty quickly -- I think it just needs to be special cased in the cost tracker to be "always free", rather than introducing a new vote-able function.

Reply:
https://github.com/blockstack/stacks-blockchain/pull/2751

Reply:
OK. If your change is in first I can update this.